zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -353,34 +346,36 @@ pub(crate) fn send_multi_step_proposed_transfer<T: ShieldedPoolTester>() {
             TransparentAddress::PublicKeyHash(data) => Address::Tex(data),
             _ => unreachable!(),
         };
-        let request = zip321::TransactionRequest::new(vec![Payment::without_memo(
-            tex_addr.to_zcash_address(&st.network()),
-            amount,
-        )])
-        .unwrap();
+        let change_memo = None;
```

Comment:
As of this commit, change memos were not correctly handled in ZIP 320 proposals. This is fixed in the next commit.